### PR TITLE
Add new Kubernetes associated resources endpoints

### PR DESCRIPTION
**Pull request by andrewsomething**
https://developers.digitalocean.com/documentation/changelog/api-v2/doks-cascading-deletes/

### CODE REVIEW

File: specification/resources/kubernetes/models/associated_kubernetes_resources.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+type: object
+description: An objects containing the IDs of resources associated with a
```
Comment (bentranter) [inline specification/resources/kubernetes/models/associated_kubernetes_resources.yml:2]: Should this be,
```suggestion
description: An object containing the IDs of resources associated with a
```

**Review by bentranter — COMMENTED** [review]
One potential typo, other than that LGTM!

File: specification/resources/kubernetes/models/associated_kubernetes_resources.yml (hunk 1 above)
Comment (andrewsomething) [reply specification/resources/kubernetes/models/associated_kubernetes_resources.yml:2]: Good catch!

**Review by bentranter — APPROVED** [review]
LGTM!

**Pull request merged**